{
  Object result=null;
  try {
    result=m.invoke(obj,args);
  }
 catch (  Throwable t) {
    log.ignoringException(m.getName(),t.getMessage(),t.getCause());
  }
 finally {
  }
  return result;
}
